minutes/2023/230531Minutes.md

```diff
@@ -0,0 +1,42 @@
+# Minutes May 31st 2023
+
+## Attendees
+- Valentina Hurtado-McCormick
+- Martino Sorbaro
+- Samantha Ahern
+- Toby Hodges
+
+## Agenda items
+
+### Relationship between CAC and LPGC
+The boundary of competences between CAC and LPGC needs to be defined, and this relationship should be publicly clarified.
+We could have another rubric for CAC vs LPGC invovement, or we could add a little bit of material on the CAC vs Maintainer rubric, or we could link to our charter.
```

Comment:
The point here is to avoid redundancy (as is often the case for Carpentries material).